### ISSUE TITLE
Can't add group permissions to groups

### ISSUE THREAD

**Issue by newswangerd** [ansible/galaxy_ng#448]
##### SUMMARY
Adding add_group, view_group, delete_group, change_group, permissions to a group on the API fails with a 500 error.

##### STEPS TO REPRODUCE
Go to the group management page, open a group to edit, add all the group permissions and click save. The 

The api throws 500 errors for each permission
![Screen Shot 2020-09-15 at 8 46 11 AM](https://user-images.githubusercontent.com/6063371/93211973-f7bb6b00-f72f-11ea-9efa-4e586b89db7f.png)

##### EXPECTED RESULTS

Group permissions should be added to the group successfully.

##### ACTUAL RESULTS

```
Traceback:

File "/venv/lib64/python3.6/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/venv/lib64/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/venv/lib64/python3.6/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/venv/lib64/python3.6/site-packages/django/views/decorators/csrf.py" in wrapped_view
  54.         return view_func(*args, **kwargs)

File "/venv/lib64/python3.6/site-packages/rest_framework/viewsets.py" in view
  114.             return self.dispatch(request, *args, **kwargs)

File "/venv/lib64/python3.6/site-packages/rest_framework/views.py" in dispatch
  505.             response = self.handle_exception(exc)

File "/venv/lib64/python3.6/site-packages/rest_framework/views.py" in handle_exception
  465.             self.raise_uncaught_exception(exc)

File "/venv/lib64/python3.6/site-packages/rest_framework/views.py" in raise_uncaught_exception
  476.         raise exc

File "/venv/lib64/python3.6/site-packages/rest_framework/views.py" in dispatch
  502.             response = handler(request, *args, **kwargs)

File "/venv/lib64/python3.6/site-packages/pulpcore/app/viewsets/user.py" in create
  133.         permission = self.get_model_permission(request)

File "/venv/lib64/python3.6/site-packages/pulpcore/app/viewsets/user.py" in get_model_permission
  101.         return get_object_or_404(Permission, **data)

File "/venv/lib64/python3.6/site-packages/django/shortcuts.py" in get_object_or_404
  93.         return queryset.get(*args, **kwargs)

File "/venv/lib64/python3.6/site-packages/django/db/models/query.py" in get
  412.             (self.model._meta.object_name, num)

Exception Type: MultipleObjectsReturned at /api/automation-hub/_ui/v1/groups/1/model-permissions/
Exception Value: get() returned more than one Permission -- it returned 2!

```

Pulpcore issue: https://pulp.plan.io/issues/7541

**Comment by chouseknecht** [comment]
https://github.com/pulp/pulpcore/pull/923

**Comment by newswangerd** [comment]
https://github.com/pulp/pulpcore/pull/923 has been merged and released. This got fixed as part of our upgrade to pulp 3.7.0
